SIPS/sip-15.md

```diff
@@ -0,0 +1,73 @@
+---
+sip: 15
+title: Snaps Home Page
+status: Draft
+discussions-to: https://github.com/MetaMask/SIPs/discussions/115
+author: Frederik Bolding (@frederikbolding)
+created: 2023-10-26
+---
+
+## Abstract
+
+This SIP proposes a new API that allows snaps to surface a static UI to display data that may be useful to users of the snap. This proposal outlines some of the details around this feature.
+
+## Motivation
+
+Snaps currently tend to leverage separate websites for showing details relating to the status of the snap. For example, snaps that manage new blockchain accounts for users have to rely almost exclusively on these to show balances on new kinds of networks. This proposal aims to start improving on this by giving snaps a surface directly in the client where they can render custom UI. 
+
+This new surface differs from the existing custom UI surfaces as it isn't reactionary, instead of being triggered by an RPC call or a transaction being confirmed the user can choose to view the home page screen at any time.
```

Comment:
One could argue that it is reactionary, it is reacting to the user activating the page by opening it within the extension, or if the Snap deep links to this page from somewhere else. Every entry point is reactionary in some way. You could say it differs in that it is directly accessible by the user via the extension UI.

Reply:
Maybe not reactionary to user input? Not sure if a view is considered input

Reply:
I guess that is true, I was merely trying to explain how this is more surface-level than other snaps features that are "part of another flow"